src/routes/notifications/routes.ts

```diff
@@ -0,0 +1,8 @@
+import { Router } from "express";
+import controller from "../../controllers/mail-controller";
+
+const router = Router();
+
+router.get('/mail', controller.getToBeSentEmail);
```

Comment:
Definitely, it makes sense to make this service as an API in long run. Good catch.
You can continue that way  if you are confortable with that.

But, as noted in the issue, it is going to be used as a cronjob. That means, the container is not going to be available all the time (like an API), but just for running a job and after that destroyed.

So I think it would make your life easier if you just call something like `ts-node src/index.ts` and it would go through the whole process. If you prefer to keep it as an API, it would make on the other hand my life easier if end up with an simple GET endpoint that does the whole process (get not sent notifications, send emails etc.).

Reply:
Thanks I will go through your cron example 

Actually, I create a generic function to send an Email function that can be called from API as well from CRON you can see in my code 

For now, if you need to send Emails on an Adhoc Basis so you can use API else cron is running every hour as described on the issue 

Actually, you are right I should create a POST request to it as I planned too create a generic API that can send email in multiple ways depending on the condition like it's also sent when seen = false only condition decided by our POST payload (This is not implemented but I planned )

you can run cron by yarn scheduler
run server by yarn serve command

Reply:
Done